src/he/ssl_ctx.c

```diff
@@ -706,3 +706,18 @@ he_return_code_t he_ssl_ctx_set_aggressive_mode(he_ssl_ctx_t *ctx) {
   ctx->use_aggressive_mode = true;
   return HE_SUCCESS;
 }
+
+
+he_return_code_t he_ssl_ctx_set_pqc_keyshares(he_ssl_ctx_t *ctx) {
+   // Return if ctx is null
+  if(!ctx) {
+    return HE_ERR_NULL_POINTER;
+  }
+
+#ifndef HE_NO_PQC
+  ctx->use_pqc_keyshares = true;
+  return HE_SUCCESS;
+#else
+  return HE_ERR_FAILED;
+#endif
```

Comment:
I suggest use #ifndef to guard the whole function, so that when `HE_NO_PQC` is defined, the symbol `he_ssl_ctx_set_pqc_keyshares` won't exist either.